### PR TITLE
[luci] Reference the right fuchsia CIPD and upload only once

### PR DESCRIPTION
**Pull request by iskakaushik**
Also only upload is os is linux

### CODE REVIEW

File: tools/fuchsia/fuchsia.cipd.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package: flutter/fuchsia
```
Comment (dnfield) [inline tools/fuchsia/fuchsia.cipd.yaml]: Rather tahn using a separate .yaml file, can we just include it all on the command line?

File: tools/fuchsia/build_fuchsia_artifacts.py [hunk 2]
```diff
@@ -91,7 +94,7 @@ def ProcessCIPDPakcage(upload, engine_version):
   cipd_yaml = os.path.join(_script_dir, 'fuchsia.cipd.yaml')
   CopyFiles(cipd_yaml, os.path.join(_bucket_directory, 'fuchsia.cipd.yaml'))
 
-  if upload:
+  if upload and IsLinux():
```
Comment (dnfield) [inline tools/fuchsia/build_fuchsia_artifacts.py]: Are we still building the mac and linux Dart packages?

Comment (iskakaushik) [reply tools/fuchsia/build_fuchsia_artifacts.py]: We will still build all the specified target. We even run `cipd pkg-build` command will still be executed. It's the else clause. We only restrict uploading the packages to linux.

File: tools/fuchsia/fuchsia.cipd.yaml (hunk 1 above)
Comment (iskakaushik) [reply tools/fuchsia/fuchsia.cipd.yaml]: Can I do it as a follow up PR?

**Comment by iskakaushik** [comment]
landed on red to fix luci

**Pull request merged**